ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -18,6 +18,15 @@
 
 ### FAN RUN CONTROL ###
 
+FAN_SS_VSC_WDPM:
+  description: "Fan with start/stop, open-loop speed control and differential pressure monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - WDPM
```

Comment:
This is typically discouraged without more context. Differential pressure across what? The zone/room to the outside?

Reply:
Hi Claire - apologies for delay only getting info back now, differential pressure is only monitoring across the fan